### ISSUE TITLE
Jfr custom power events

### ISSUE THREAD

**Issue by franz1981** [metacosm/quarkus-power#27]
I would explore to emit jfr custom events to enable users which collect them to (for example), enable jfr profiling, if the retrieved thresholds are beyond some specified value.

**Comment by metacosm** [comment]
Yes, this is on the list of things I'm planning on investigating. I just want to validate the architecture first.
